packages/big-design/src/components/ToggleButtons/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
The folder path for this component seems off `components/ToggleButtons/ToggleButton` 🤔. I would set the directory structure for this like so:
```
components/
| – ...
| – ToggleButton/
| –––– index.ts
| –––– spec.tsx
| –––– styled.tsx
| –––– ToggleButton.tsx
```
Alongside that, I would remove the `index.ts` file in favor of a `private.ts` file.

Reply:
I wasnt sure whether we wanted to keep ToggleButton at top level of components or treat it as implementation details of ToggleButtons, happy to more though.

Reply:
We should keep it at the top level for now. When we get to the group of `ToggleButtons` we can refactor, if it makes sense.